packages/patternfly-4/react-core/src/components/Alert/Alert.js

```diff
@@ -30,6 +30,8 @@ const propTypes = {
   'aria-label': PropTypes.string,
   /** Variant label text for screen readers */
   variantLabel: PropTypes.string,
+  /** A callback for when the close button is clicked (if undefined, no close button is rendered) */
+  onClose: PropTypes.func,
```

Comment:
Do we need to update the d.ts file?

Reply:
Whoops, I thought I did, but I was editing the `dist` copy instead of `src`. Will fix.